internal/appsec/remoteconfig.go

```diff
@@ -11,6 +11,8 @@ import (
 	"fmt"
 	"os"
 
+	"golang.org/x/exp/maps"
```

Comment:
This appears to be a new dependency, introduced only for `maps.Values`, which is a trivial 4-line function... Maybe you can just implement it locally instead of taking the dependency?

Reply:
done